test/builder/README.md

```diff
@@ -0,0 +1,120 @@
+# Builder package for tests
+
+This package holds `Builder` functions that can be used to create
+struct in tests with less noise.
+
+One of the most important characteristic of a unit test (and any type
+of test really) is **readability**. This means it should be easy to read
+but most importantly it should clearly show the intent of the
+test. The setup (and cleanup) of the tests should be as small as
+possible to avoid the noise. Those builders exists to help with that.
+
+There is two types of functions defined in that package :
+
+	* *Builders*: create and return a struct
+	* *Modifiers*: return a function
+	  that will operate on a given struct. They can be applied to other
+	  Modifiers or Builders.
+
+Most of the Builder (and Modifier) that accepts Modifiers defines a
+type (`TypeOp`) that can be satisfied by existing function in this
+package, from other package *or* inline. An example would be the
+following.
+
+```go
+	// Definition
+	type TaskOp func(*v1alpha1.Task)
+	func Task(name string, ops ...TaskOp) *v1alpha1.Task {
```

Comment:
Docs still refer to the old function signature.

Reply:
Ah I though I forgot one somewhere :man_facepalming: